src/components/views/elements/Dropdown.js

```diff
@@ -193,19 +191,14 @@ export default class Dropdown extends React.Component {
         this.props.onOptionChange(dropdownKey);
     }
 
-    _onInputKeyPress(e) {
-        // This needs to be on the keypress event because otherwise it can't cancel the form submission
-        if (e.key === Key.ENTER) {
-            e.preventDefault();
-        }
-    }
+    _onInputKeyDown = (e) => {
+        let handled = true;
 
-    _onInputKeyUp(e) {
         // These keys don't generate keypress events and so needs to be on keyup
         switch (e.key) {
             case Key.ENTER:
                 this.props.onOptionChange(this.state.highlightedOption);
-                // fallthrough
+            // fallthrough
```

Comment:
IDE fail? Indentation feels weird this way.

Reply:
indeed copy-paste triggered IDE auto-indent